cmd/swarmctl/cluster/update.go

```diff
@@ -136,5 +143,6 @@ func init() {
 	updateCmd.Flags().StringSlice("secret", nil, "Secret required to join the cluster")
 	updateCmd.Flags().Int64("taskhistory", 0, "Number of historic task entries to retain per slot or node")
 	updateCmd.Flags().Duration("certexpiry", 24*30*3*time.Hour, "Duration node certificates will be valid for")
+	updateCmd.Flags().StringSlice("external-ca-url", nil, "URL to one or more CFSSL CA certificate signing endpoints")
```

Comment:
I don't think that the flag should be pluralized since you would generally specify the flag once per URL:

```
... \
--external-ca-url 'https://ca1.example.com' \
--external-ca-url 'https://ca2.example.com' \
...
```

and they get appended to a list internally. This type of usage seems more natural to not pluralize the flag name.

What is weird (and poorly documented in my opinion) about the `spf13/cobra` library is that the `StringSlice` flag can either be set multiple times to append to the list but the argument is parsed as a comma-separated-value (see https://github.com/spf13/pflag/blob/master/string_slice.go#L24) and so you could also specify it like this:

```
... \
--external-ca-urls 'https://ca1.example.com,https://ca2.example.com' \
...
```

in which case it does seem more natural (to me at least) to pluralize the name of the flag since the argument does represent multiple values.

I mostly want to stick with whatever precedent there is. It looks like `secret` and `autoaccept` are also not pluralized here.